storage/persistent_storage/persistent-storage-ocs.adoc

```diff
@@ -5,91 +5,91 @@ include::modules/common-attributes.adoc[]
+|Deploying Red Hat OpenShift Data Foundation on IBM Power Systems
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/deploying_openshift_data_foundation_using_ibm_power_systems/[Deploying OpenShift Data Foundation using IBM Power Systems]
 
-|Deploying Red Hat OpenShift Container Storage on IBM Cloud
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_openshift_container_storage_using_ibm_cloud/[Deploying OpenShift Container Storage using IBM Cloud]
+|Deploying Red Hat OpenShift Data Foundation on IBM Cloud
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/deploying_openshift_data_foundation_using_ibm_cloud/[Deploying OpenShift Data Foundation using IBM Cloud]
 
-|Deploying and managing Red Hat OpenShift Container Storage on {rh-openstack-first}
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_and_managing_openshift_container_storage_using_red_hat_openstack_platform/[Deploying and managing OpenShift Container Storage 4.7 using Red Hat OpenStack Platform]
+|Deploying and managing Red Hat OpenShift Data Foundation on {rh-openstack-first}
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/deploying_and_managing_openshift_data_foundation_using_red_hat_openstack_platform/[Deploying and managing OpenShift Data Foundation 4.9 using Red Hat OpenStack Platform]
 
-|Deploying and managing Red Hat OpenShift Container Storage on {rh-virtualization-first}
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_openshift_container_storage_using_red_hat_virtualization_platform/index[Deploying and managing OpenShift Container Storage 4.7 using Red Hat Virtualization Platform]
+|Deploying and managing Red Hat OpenShift Data Foundation on {rh-virtualization-first}
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/deploying_openshift_data_foundation_on_vmware_vsphere/[Deploying and managing OpenShift Data Foundation 4.9 using Red Hat Virtualization Platform]
 
-|Deploying Red Hat OpenShift Container Storage on VMware vSphere clusters
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html/deploying_openshift_container_storage_on_vmware_vsphere/[Deploying OpenShift Container Storage 4.7 on VMware vSphere]
+|Deploying Red Hat OpenShift Data Foundation on VMware vSphere clusters
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/deploying_openshift_container_storage_on_vmware_vsphere/[Deploying OpenShift Data Foundation 4.9 on VMware vSphere]
```

Comment:
This link gives a 404; should it be this? https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/deploying_openshift_data_foundation_on_vmware_vsphere

Reply:
Addressed